content/cloud-computing/concepts/shared-responsibility-model/shared-responsibility-model.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: 'Shared Responsibility Model'
+Description: 'The shared responsibility model means that cloud providers maintain some responsibility for security, but some security responsibilities are borne by the customer.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Cloud Computing'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+The **shared responsibility model** means that cloud providers maintain some responsibility for security, but some security responsibilities are borne by the customer. In order to provide a secure environment using cloud services, the customer must be cognizant of where the provider's responsibilities end, and where their own responsibilities start.
+
+The line between customer and provider responsibilities can vary between service providers, and can change depending on what type of service is being provided. For instance, the responsibilities may differ between [infrastructure as a service (IaaS)](https://www.codecademy.com/resources/docs/cloud-computing/iaas) and [platform-as-a-service (PaaS)](https://www.codecademy.com/resources/docs/cloud-computing/paas).
```

Comment:
This issue seems to be in other Cloud entries, but we should settle on whether to hyphenate the expanded entry name or not (i.e., `platform-as-a-service (PaaS)` or `platform as a service (PaaS)`) for this one.